matrix_sdk_crypto/src/key_request.rs

```diff
@@ -314,6 +314,10 @@ impl KeyRequestMachine {
             }
             // We ignore cancellations here since there's nothing to serve.
             Action::CancelRequest => return Ok(()),
+            ref action => {
```

Comment:
Could match on `&event.content.action` above and remove this `ref`.

Reply:
Done